src/api/parser/src/lib/elastic.js

```diff
@@ -1,9 +1,16 @@
+/*
+ * NOTE: This has been ported to ~/src/api/search
+ * Future updates will remove the codebase below
+ * and point towards `service.docker.localhost` for querying.
+ */
+
 const { ELASTIC_URL, ELASTIC_PORT } = process.env;
```

Comment:
We don't want any of this.  Use elastic/mock from Satellite.

cc @rclee91, @JiaHua-Zou

Reply:
I don't think this file is used, I changed the import in `indexer.js` to `const { Elastic } = require('@senecacdot/satellite')`, it should take new changes in Satellite